src/rendering/renderers/shared/system/AbstractRenderer.ts

```diff
@@ -107,6 +107,8 @@ export class AbstractRenderer<PIPES, OPTIONS extends PixiMixins.RendererOptions,
     public view: ViewSystem;
     public textureGenerator: GenerateTextureSystem;
 
+    protected _initOptions: OPTIONS = {} as OPTIONS;
```

Comment:
Have added this purely for the tests to access as I wasn't sure of another possible way.

Reply:
as we are creating the renderer in the tests, do we not know this externally?